src/test/UnionStruct.fbs

```diff
@@ -0,0 +1,15 @@
+struct StructA {
```

Comment:
Can you add a comment to this file describing what you are trying to accomplish here?

Reply:
Added - hopefully it is clear from the comment what the intent of the file is.